gravitee-apim-console-webui/Dockerfile

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+FROM graviteeio/nginx:1.23
+LABEL maintainer="contact@graviteesource.com"
+
+COPY ./dist /usr/share/nginx/html/
+ADD docker/config /etc/confd
```

Comment:
### :warning: Aqua detected misconfiguration in your code  
**Misconfiguration ID:** AVD-DS-0005   
**Check Name:** ADD instead of COPY   
**Severity:** LOW   
**Message:** Consider using 'COPY docker/config /etc/confd' command instead of 'ADD docker/config /etc/confd'  
**Resolution:** Use COPY instead of ADD  
[This comment was created by Aqua Pipeline]  
  
Read more at https://avd.aquasec.com/misconfig/ds005